scripts/safe_template_linter.py

```diff
@@ -1037,33 +1183,69 @@ def _process_os_walk(starting_dir, template_linters, options, out):
         _process_current_walk(current_walk, template_linters, options, out)
 
 
+def _parse_arg(arg, option):
```

Comment:
I think this function means it's time to use a real parser.  argparse is simple and in the stdlib if you don't want to go all the way to click.